src/ObjectUtils/CoffFile.cpp

```diff
@@ -134,16 +132,11 @@ std::optional<SymbolInfo> CoffFileImpl::CreateSymbolInfo(
   symbol_info.set_address(symbol_virtual_address);
 
   // The COFF symbol table doesn't contain the size of symbols. Set a placeholder for now.
-  symbol_info.set_size(kUnknownSymbolSize);
+  symbol_info.set_size(ObjectFile::kUnknownSymbolSize);
```

Comment:
I think that before with the logic being completely local to `CoffFileImpl` it was clearer how this was a placeholder to be replaced by `DeduceDebugSymbolMissingSizes`. Now this logic is spread out and it's less clear that setting this value is fine because `DeduceDebugSymbolMissingSizes` is going to be called afterwards (we don't want any symbol to have size `kUnknownSymbolSize` in the end). At least expanding the comment mentioning that this placeholder is for the purpose of `DeduceDebugSymbolMissingSizes` could help.

Reply:
Extended the comment.